src/AccessibilityInsights/Modes/SnapshotModeControl.xaml

```diff
@@ -39,7 +39,7 @@
                         ResizeDirection="Columns"
                         BorderBrush="Gray" BorderThickness="1"
                         AutomationProperties.Name="{x:Static properties:Resources.gsMidAutomationPropertiesName}"
-                        AutomationProperties.HelpText="{x:Static properties:Resources.HandleFailedSelectionResetConnectionLostMessage}"/>
+                        AutomationProperties.HelpText="{x:Static properties:Resources.gsMidAutomationPropertiesHelpText}"/>
```

Comment:
This is the actual bug. The rest is just cleaning up the duplicate string resource